modules/table/src/test/java/org/apache/ignite/internal/table/distributed/replication/PartitionReplicaListenerTest.java

```diff
@@ -3279,13 +3279,13 @@ void testBuildIndexReplicaRequestWithoutRwTxOperations() {
     @ValueSource(booleans = {false, true})
     void testBuildIndexReplicaRequest(boolean failCmd) {
         var continueNotBuildIndexCmdFuture = new CompletableFuture<Void>();
-        var buildIndexCommandFuture = new CompletableFuture<BuildIndexCommand>();
+        var buildIndexCommandFuture = new CompletableFuture<BuildIndexCommandV2>();
 
         when(mockRaftClient.run(any())).thenAnswer(invocation -> {
             Command cmd = invocation.getArgument(0);
 
-            if (cmd instanceof BuildIndexCommand) {
-                buildIndexCommandFuture.complete((BuildIndexCommand) cmd);
+            if (cmd instanceof BuildIndexCommandV2) {
```

Comment:
I suggest return the old value `BuildIndexCommand`, since essentially nothing has changed.